pyproject.toml

```diff
@@ -26,48 +26,48 @@ celery = {extras = ["redis"], version = "^5.1.2"}
 python = "^3.8.0"
 fastapi = "^0.68.0"
 # Per the `uvicorn docs <https://www.uvicorn.org/#quickstart>`_, install the standard (as opposed to minimal) uvicorn dependencies.
-uvicorn = {extras = ["standard"], version = "^0.14.0"}
+uvicorn = {extras = ["standard"], version = "^0.17.0"}
 # See the `poetry docs <https://python-poetry.org/docs/dependency-specification/#using-environment-markers>`_.
-gunicorn = {version = "^20.1.0", markers = "sys.platform != 'win32'"}
-Jinja2 = "^3.0.1"
-aiofiles = "^0.6.0"
-alembic = "^1.4.3"
-python-dateutil = "^2.8.1"
-fastapi-login = "1.6.3"
```

Comment:
I'm not sure how much of fastapi-login we even use anymore....  But I'll need to test this carefully, login bumps have caused me too many headaches already.

Reply:
Sounds good. I agree that manual testing is always important, especially since our test coverage is < 50%.

Reply:
I did some quick manual testing -- everything works for me. I'm using the web2py-based login, since we rarely log in using the BookServer.